README.md

```diff
@@ -327,6 +361,7 @@ You can find plenty of sample code that illustrates management scenarios (69+ en
     <td>App Service - Web Apps on <b>Linux</b></td>
     <td><ul style="list-style-type:circle">
 <li><a href="https://github.com/Azure-Samples/app-service-dotnet-manage-web-apps-on-linux">Manage Web apps</a></li>
+<li><a href="https://github.com/Azure-Samples/app-service-dotnet-deploy-image-from-acr-to-linux">Deploy a container image from Azure Container Registry to Linux containers</a></li>
```

Comment:
Sample not available at this time; line 364 should be remove/commented out via <!